Doc/library/concurrent.futures.rst

```diff
@@ -257,6 +257,10 @@ ProcessPoolExecutor Example
        1099726899285419]
 
    def is_prime(n):
+       if n == 1:
```

Comment:
I suggest n < 2 :-)

Reply:
Maybe but I conider that one should not give us a negative number here as a parameter, it's out of range. It's like giving a float or a list or anything else.

Reply:
0 is positive and is not prime :-)

Reply:
ok ok þ